### PR TITLE
Add total grade and context to save user for questionnarie

### PR DESCRIPTION
**Pull request by leechechik**

### CODE REVIEW

File: src/components/UserSelection.jsx [hunk 1]
```diff
@@ -1,23 +1,48 @@
...
+import { myFirebase } from "./Firebase";
+import MyContext from '../MyContext';
+import { useHistory } from "react-router-dom";
 
-const UserSelection = (props) => {
+export default (props) => {
+    let history = useHistory();
+
+    const anonymouslyAuth = (context) => {
+      console.log('before..');
```
Comment (shakednai1) [inline src/components/UserSelection.jsx:12]: please remove

File: src/components/UserSelection.jsx [hunk 2]
```diff
@@ -1,23 +1,48 @@
...
+      myFirebase.auth().signInAnonymously()
+        .then(function() {
+          console.log('Logged in as Anonymous!');
```
Comment (shakednai1) [inline src/components/UserSelection.jsx:15]: please remove

File: src/components/UserSelection.jsx [hunk 3]
```diff
@@ -1,23 +1,48 @@
...
+          myFirebase.auth().onAuthStateChanged(function(user) {
+            if (user) {
+              var uid = user.uid;
+              console.log(uid);
```
Comment (shakednai1) [inline src/components/UserSelection.jsx:19]: same

File: src/components/UserSelection.jsx (hunk 1 above)
Comment (leechechik) [reply src/components/UserSelection.jsx]: thanks, removed it

**Pull request merged**